index.js

```diff
@@ -0,0 +1,104 @@
+require('dotenv').config();
+const express = require('express');
+const cors = require('cors');
+const bcrypt = require('bcryptjs');
+const jwt = require('jsonwebtoken');
+const jwtSecret = process.env.JWT_SECRET || 'add a secret to your .env file with this key';
```

Comment:
Good work implementing this into your project. I hope you created a `.env` file with your secret in it and listed the file in your `.gitignore`.